index.bs

```diff
@@ -1324,9 +1324,10 @@ interface AudioContext : BaseAudioContext {
 </xmp>
 
 An {{AudioContext}} is said to be <dfn>allowed to start</dfn> if the user agent
-allows the context state to transition from "{{AudioContextState/suspended}}" to "{{AudioContextState/running}}". A user
-agent may require the actual audio production from an {{AudioContext}} to be
-<a>triggered by user activation</a> (as described in [[HTML]]).
+allows the context state to transition from "{{AudioContextState/suspended}}" to
+"{{AudioContextState/running}}". A user agent may delay this initial transition,
+to allow it only when the {{AudioContext}}'s [=relevant global object=] has
+[=sticky activation=] or [=transient activation=].
```

Comment:
The first condition is enough here because having transient activation implies having sticky activation (and not vice versa).